service/controller/chart/v1/resource/status/create.go

```diff
@@ -76,7 +76,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 		currentCR.Status = desiredStatus
 
-		_, err = r.g8sClient.ApplicationV1alpha1().Charts(cr.Namespace).UpdateStatus(currentCR)
+		_, err = r.g8sClient.ApplicationV1alpha1().Charts(cr.Namespace).Update(currentCR)
```

Comment:
If I change this to Update it works. With UpdateStatus it fails with the error below. I guess its related to my change but I don't get how.

It happens in both my test cluster and locally so I don't think its RBAC related.

```
D 08/29 10:23:16 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/kube-state-metrics statusv1 getting status for release `kube-state-metrics` | chart-operator/service/controller/chart/v1/resource/status/create.go:22 | controller=chart-operator-chart | event=update | loop=0 | version=26683
D 08/29 10:23:16 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/kube-state-metrics statusv1 setting status for release `kube-state-metrics` status to `DEPLOYED` | chart-operator/service/controller/chart/v1/resource/status/create.go:69 | controller=chart-operator-chart | event=update | loop=0 | version=26683
E 08/29 10:23:16 /apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/kube-state-metrics statusv1 stop reconciliation due to error | operatorkit/controller/controller.go:370 | controller=chart-operator-chart | event=update | loop=0 | version=26683
	/Users/rossf7/.gvm/pkgsets/go1.12.5/global/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/operatorkit/controller/controller.go:530:
	/Users/rossf7/.gvm/pkgsets/go1.12.5/global/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/metricsresource/resource_wrapper.go:39:
	/Users/rossf7/.gvm/pkgsets/go1.12.5/global/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/retryresource/resource_wrapper.go:79:
	/Users/rossf7/.gvm/pkgsets/go1.12.5/global/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/operatorkit/resource/wrapper/retryresource/resource_wrapper.go:67:
	/Users/rossf7/.gvm/pkgsets/go1.12.5/global/src/github.com/giantswarm/chart-operator/service/controller/chart/v1/resource/status/create.go:81:
	the server could not find the requested resource (put charts.application.giantswarm.io kube-state-metrics)
```